frontend/src/pages/[username]/[projectName]/IBOM.js

```diff
@@ -74,40 +63,59 @@ const IBOM = ({ html, pcbData }) => {
   `
 
   useEffect(() => {
+    const config = {
+      dark_mode: false,
+      show_pads: true,
+      show_fabrication: false,
+      show_silkscreen: true,
+      highlight_pin1: true,
+      redraw_on_drag: true,
+      board_rotation: 0,
+      checkboxes: 'Sourced,Placed',
+      bom_view: 'left-right',
+      layer_view: 'FB',
+      extra_fields: [],
+    }
+
     window.config = config
-    setReady(true)
   }, [])
 
-  if (ready) {
-    return (
-      <Page contentFullSize>
-        <NextHead>
-          <script
-            type="text/javascript"
-            src="/static/IBOM/index.js"
-            id="IBOM_script"
-          />
-          <script type="text/javascript">{initScript}</script>
-          <link rel="stylesheet" href="/static/IBOM/index.css" />
-        </NextHead>
+  return (
+    <Page title={title} contentFullSize>
+      <Script
+        src="/static/IBOM/index.js"
+        id="IBOM_script"
+        onLoad={() => {
+          setReady(true)
+          return
+        }}
+      />
+      <Script>{initScript}</Script>
+      <NextHead>
+        {/* Styles for IBOM targets uses non pure selectors, e.g., `:root`, importing its style as
+         * a  module doesn't work. The other option was to add the style to `_app.scss`
+         * which will load on visiting any page.
+         */}
+        {/* eslint-disable-next-line @next/next/no-css-tags */}
```

Comment:
Does `eslint-disbable` have to be a separate comment?

Reply:
Yes